common/test/acceptance/pages/studio/users.py

```diff
@@ -8,19 +8,6 @@
 from . import BASE_URL
 
 
-def wait_for_ajax_or_reload(browser):
```

Comment:
Why did you delete this method and replace all of its invocations with `wait_for_ajax`?